### PR TITLE
doc: add button to copy code blocks in Dune manual

### PR DESCRIPTION
**Pull request by MisterDA**
This is imo a nice usability improvement ;-)

Using sphinx-copybutton <https://sphinx-copybutton.readthedocs.io/en/latest/>.

Hope I've setup nix flakes correctly. Building works locally but I haven't tested nix stuff.

### CODE REVIEW

**Review by christinerose — APPROVED** [review]

File: doc/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+sphinx >= 4.5.0
+sphinx_rtd_theme >= 1.0.0
+sphinx-copybutton >= 0.5.0
```
Comment (Alizter) [inline doc/requirements.txt:3]: Could you point the "requirements" in the doc makefile job here too?

**Review by Alizter — APPROVED** [review]
Great stuff!

File: doc/requirements.txt (hunk 1 above)
Comment (MisterDA) [reply doc/requirements.txt:3]: Done!

**Comment by emillon** [comment]
docs are deployed separately from releases so I'm removing the milestone

**Pull request merged**